stdlib.colours

```diff
@@ -0,0 +1,17 @@
+################################################################################
+#
+# stdlib.colours
+#
+################################################################################
+
+[colours]
+## name = <fg color>,[<bg color>],[<fg mode>]
```

Comment:
I'd make the commas in `stdlib.colours`optional unless followed by a further colour name